src/modules/Experience/scenes/WelcomeScene/components/Message/Message.tsx

```diff
@@ -93,7 +93,7 @@ const Message = () => {
           )}
         </Box>
       </Flex>
-      <Clouds material={THREE.MeshBasicMaterial}>
+      {/* <Clouds material={THREE.MeshBasicMaterial}>
```

Comment:
So what is happening with this clouds??  

Are they in or out?? XD 


Over all did you test your solution mobile tablet and desktop?

Reply:
The clouds make the website break for me.
I made sure that solution is working on mobile and other devices in the next comit.

Reply:
So there are no clouds anymore ???

Reply:
We will keep them, but will optimize them in a separate ticket.